bundles/org.openhab.ui/web/src/pages/settings/items/item-details.vue

```diff
@@ -184,9 +184,18 @@ export default {
       this.$store.dispatch('stopTrackingStates')
     },
     load () {
-      this.$oh.api.get(`/rest/items/${this.itemName}?metadata=semantics,${MetadataNamespaces.map((n) => n.name).join(',')}`).then((data) => {
-        this.item = data
-        this.iconUrl = (localStorage.getItem('openhab.ui:serverUrl') || '') + '/icon/' + this.item.category + '?format=svg'
+      let metadataNamespacesList = MetadataNamespaces.map((n) => n.name).join(',')
+      this.$oh.api.get(`/rest/items/${this.itemName}/metadata/namespaces`).then((itemNamespaces) => {
+        this.$oh.api.get(`/rest/items/${this.itemName}?metadata=${itemNamespaces.join(',')}`).then((data) => {
+          // map the labels of the known namespaces or add "custom:" to the user defined namespaces
+          Object.keys(data.metadata).forEach(namespace => {
+            let rs = MetadataNamespaces.find(ns => ns.name === namespace)
+            data.metadata[namespace].id = namespace
+            data.metadata[namespace].label = (rs === undefined) ? 'custom: ' + namespace : rs.label
```

Comment:
Not sure if marking a namespace as custom as each of them is custom, I'd rather say others are "predefined" or "standard". Maybe not making any extra text there would be better because any prefix or suffix brings another implication - a translation.

Reply:
I hear what you are saying though I still think some marking makes sense as you are _creating_ it as a _custom_ name as well. These are kind of special. 

<img width="316" alt="image" src="https://user-images.githubusercontent.com/5937600/210909456-99a8405b-679c-4d27-bb67-1a257a166674.png">

I was even thinking if we should kind of sort them to the bottom. With some effort we could maybe mark them with some kind of icon though I probably have to research bit how to do that here in vue.

Reply:
> // map the labels of the known namespaces or add "custom:" to the user defined namespaces

Yeah I agree with @splatch here - I don't support the UI making assumptions or restrictions or enforcing prefixes on the namespaces you can edit.

Reply:
I am not sure if I brought across my intent. My idea was to somehow make visible the namespaces that are custom created by the user and the ones that are known by openHAB. I was't giving them prefixes in the name I only presented that in the UI. What about marking them with an icon or just separate them by a line?

<img width="790" alt="image" src="https://user-images.githubusercontent.com/5937600/211192711-0c72da60-2b4e-4a85-979e-e49b032464d7.png">

Reply:
I support the idea of somehow making visible that a namespaces is a custom user namespace that is not directly linked to any openHAB functionality (by this I mean for example the expire metadata that is read and processed by core). 

Prefixing what is shown IMO can be confusing for the end user, I’d rather put in a line together with some wording like „User defined metadata“.

Reply:
I optimised the loading via the ".*" param and separated standard and custom namespaces by a horizontal line (and removed the icon). See edited description of the PR.